dissect/target/plugins/os/unix/linux/proc.py

```diff
@@ -529,19 +527,19 @@ def _boottime(self) -> int:
 
         Used internally to determine process start- and runtimes.
         """
-        for line in self.target.fs.get("/proc/stat").open().readlines():
-            if not line.startswith(b"btime"):
+        for line in self.target.fs.path("/proc/stat").open("rt").readlines():
+            if not line.startswith("btime"):
                 continue
 
             return int(line.split()[1])
 
-    def get(self, path) -> FilesystemEntry:
-        """Returns a FilesystemEntry relative to this process."""
-        return self.entry.get(path)
+    def get(self, path: fsutil.TargetPath) -> fsutil.TargetPath:
+        """Returns a TargetPath relative to this process."""
+        return self.entry.joinpath(path)
```

Comment:
```suggestion
    def get(self, path: pathlib.Path) -> pathlib.Path:
        """Returns a ``Path`` relative to this process."""
        return self.entry.joinpath(path)
```
I don't think there's any specific reason to note that it's a `TargetPath`, so just use the more generic `Path`.